docs/diagrams/PinCommand.puml

```diff
@@ -0,0 +1,40 @@
+@startuml
```

Comment:
There is an empty square bracket in this UML diagram. Is there meant to be something there or is it unintentionally added? Perhaps you can review this part of the diagram.
![Screenshot 2020-10-30 at 1 10 55 PM](https://user-images.githubusercontent.com/60461351/97662252-7a667280-1ab1-11eb-9581-035d5c612fc2.png)